plugins/techdocs/src/reader/components/TechDocsNotFound.tsx

```diff
@@ -37,7 +38,7 @@ export const TechDocsNotFound = ({ errorMessage }: Props) => {
 
   return (
     <ErrorPage
-      status="404"
+      status={statusCode ? statusCode.toString() : '404'}
```

Comment:
> Note: I was unable to figure out how to pass the HTTP error code from the API back into the component, so while the React UI component COULD say ERROR 500, it doesn't yet. Happy for feedback on that, or to iterate later.

I thought the ErrorPage from `@backstage/core` is responsible for the error code. Sorry, I don't think I understand "it doesn't yet" part!

Reply:
Yeah sorry, this component `<TechDocsNotFound>` receives the status, and so it gets passed into the error code.  The challenge I had was elsewhere... I'll separately comment where exactly I meant.